src/sentry/static/sentry/app/views/releasesV2/list/introBanner.tsx

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import {t} from 'app/locale';
-import theme from 'app/utils/theme';
 import Banner from 'app/components/banner';
+import Button from 'app/components/button';
 import localStorage from 'app/utils/localStorage';
+import space from 'app/styles/space';
+
+import backgroundLighthouse from '../../../../images/spot/background-lighthouse.svg';
```

Comment:
@doralchan should I add a webpack alias for the images directory? any good ideas for an alias? `sentry-images`?

Reply:
That would be great!